src/components/slideshow/slideshow.js

```diff
@@ -501,9 +572,22 @@ export default function (options) {
     function showOsd() {
         const bottom = dialog.querySelector('.slideshowBottomBar');
         if (bottom) {
-            slideUpToShow(bottom);
-            startHideTimer();
+            slideToShow(bottom, 'down');
         }
+
+        const exit = dialog.querySelector('.btnSlideshowExit');
+        if (exit) slideToShow(exit, 'up');
+
+        // When autoplaying, we don't put back the arrows
+        if (!autoPlayActive) {
+            const left = dialog.querySelector('.btnSlideshowPrevious');
+            if (left) slideToShow(left, 'left');
+
+            const right = dialog.querySelector('.btnSlideshowNext');
+            if (right) slideToShow(right, 'right');
+        }
```

Comment:
The nice thing about these buttons is that you can manually flip slides and stop autoplay at the same time. I think this is very helpful.
The same thing happens when you swipe slides on mobile.

Maybe left them in place even with autoplay?

Reply:
The main reason I put this in is because someone asked if it was possible to go "full" fullscreen for a photo frame. If I remember correctly, it was @MrTimscampi who told me to hide them in autoplay, but maybe we could put them back?

Reply:
I think it's enough that they hide on user idle.

Reply:
Fair to me, fixed!